### PR TITLE
project-start

### PR DESCRIPTION
**Pull request by LarynaB**

### CODE REVIEW

File: database/authentication/auth-router.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          res.status(200).json({ message: 'logged out' });
+        }
+      });
+    } else {
+      res.status(200).json({ message: 'logged out' });
+    }
+  });
+
+//export router!
+module.exports = router;
```
Comment (shawnantonucci) [inline database/authentication/auth-router.js:67]: loving the comments short and simple and helpful later on

File: database/authentication/restricted-middleware.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+//             res.status(401).json({ message: "You shall not pass!!" });
+//           }
+//         })
+//         .catch(error => {
+//           res.status(500).json(error);
+//         });
+//     } else {
+//       res.status(401).json({ message: "Please provide credentials" });
+//     }
+//   }
```
Comment (shawnantonucci) [inline database/authentication/restricted-middleware.js:47]: nice refactor

**Review by shawnantonucci — APPROVED** [review]
Nice work on the project Laryna. great job on refactoring your authorization!

**Comment by LarynaB** [comment]
Thx

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Pull request merged**